src/app/delivery/delivery.controller.ts

```diff
@@ -17,36 +20,52 @@ export class DeliveryController {
   constructor(private readonly deliveryService: DeliveryService) {}
 
   @Get()
-  async getDeliveries() {
-    return;
+  async getDeliveries(): Promise<DeliveryPreviewResponse[]> {
+    const deliveries = await this.deliveryService.getDeliveries();
+    return deliveries.map(
+      (deliveryService) => new DeliveryPreviewResponse(deliveryService),
```

Comment:
여기 오타있네요. 수정해주세용.
deliveryService라는 이름의 변수를 생성해서 위 deliveryService 변수와 충돌날 수 있습니다.
```suggestion
    return deliveries.map(
      (delivery) => new DeliveryPreviewResponse(delivery),
```